sass/directives/_visual-guides.scss

```diff
@@ -0,0 +1,128 @@
+/* A simple, center dividing column guide / line. See /recruiting-solutions/small-business-subscription-plans-and-pricing
+ * The guide can extend out of its relatively positioned parent container on the bottom and top
+ * This guide does not include steps. It is only a center line.
+ * You must have a parent container (relatively positioned) for this to work properly.
+ */
+@mixin visual-guide-divider($extend-top: 0px, $extend-bottom: 0px) {
+
+  @if $extend-top != 0px {
+    bottom: -($extend-bottom);
+    top: -($extend-top);
+  } @else {
+    bottom: $extend-bottom;
+    top: $extend-top;
+  }
+
+  left: 0;
+  margin: auto;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  width: 50px;
+
+  &:before, &:after {
+    content: '';
+    background-color: white;
+    border: $base-border;
+    border-radius: 50%;
+    display: block;
+    height: $visual-divider-endpoint-size;
+    margin: 0 auto;
+    position: relative;
+    width: $visual-divider-endpoint-size;
+  }
+
+  &:after {
+    background-color: $grey-lighter;
+  }
+
+  .visual-guide__divider {
+    background-color: $base-border-primary-color;
+    height: 100%;
+    margin: 0 auto;
+    width: 1px;
+  }
+}
+
+/* For <li> elements only within a <ol>
+ * Parameter takes in a CSS counter that must be set / reset elsewhere
+ * Each row (your <li> content) consists of a block of text, and an image, with a center gutter.
+ * Modifiers include left properties to right justify content blocks (not text) along the center gutter.
+ * TODO: Concerning guides with no steps
+ */
+@mixin visual-guide-row ($counter) {
+  display: flex;
+  position: relative;
+
+  .visual-guide__content, .visual-guide__image {
+    display: flex;
+    width: 50%;
+
+    &:first-of-type {
+      margin-right: $base-spacing-large;
+    }
+
+    &:last-of-type {
+      margin-left: $base-spacing-large;
+    }
+  }
+
+  .visual-guide__content {
+    &.guide__content--left {
+      justify-content: flex-end;
+      padding-left: $type-margin;
```

Comment:
Just use `20px` here instead of the variable

We don't currently have a variable for the 20px of padding that we have on the outside of `.container`s (maybe create one? We haven't had a lot of need for it until more recently). That's what you would use here.

You don't want to use `type-margin` because it refers to the amount of space between lines of text, usually between a header and paragraph. If we decide to change our headers and paragraphs you wouldn't want the `padding-left` to change too b/c it's not related to text, it's related to the `.container` class